connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -159,6 +230,51 @@ export async function githubReposSyncWorkflow(
   await githubSaveSuccessSyncActivity(dataSourceConfig);
 }
 
+export async function githubSyncReposWorkflow(
+  dataSourceConfig: DataSourceConfig,
+  connectorId: ModelId,
```

Comment:
did we say we need to copy if the workflow does not change, but only the child workflow?
i don't recall (but I'd support if we're unsure :+1: )